tests/__helpers__/schemaRepositoryMock.ts

```diff
@@ -7,8 +8,37 @@ export const schemaRepositoryMock = () => {
   const disableSchemaSpy = jest.spyOn(schemaRepository, 'disable')
     .mockImplementation(() => Promise.resolve({ affectedRecords: 1 }));
 
+  const fetchSchemaByNameSpy = jest.spyOn(schemaRepository, 'fetchSchemaByName')
+    .mockImplementation(() => Promise.resolve(undefined));
+
+  const createSchemaSpy = jest.spyOn(schemaRepository, 'createSchema')
+    .mockImplementation((_sdk, name, description) => Promise.resolve(createEmptySchema(name, description)));
```

Comment:
Can we just provide SDK as null here? Then we can check 'hasBeenCalledWith'

Reply:
It is provided as null, I just need to name the parameter here because I need the second and third parameter.